src/pages/studyView/charts/violinPlotTable/StudyViewViolinPlot.tsx

```diff
@@ -0,0 +1,233 @@
+import * as React from 'react';
```

Comment:
maybe for next iteration but it would we nice to implement this using functional component.  good opportunity